pages/common/nkf.md

```diff
@@ -0,0 +1,27 @@
+# nkf
+
+> Convert character code and new line code.
+
+- Stdout utf-8 code:
+
+`nkf -w {{path/to/file.txt}}`
+
+- Stdout shift_jis code:
+
+`nkf -s {{path/to/file.txt}}`
+
+- Overwrite as utf-8 code:
```

Comment:
Convert to UTF-8 encoding and overwrite the file.

Also, I see --overwrite has a suffix value in the man page. Is that not true ?

Reply:
Possible. But it would be good to confirm this.